e2e/packages/custom-render/types.ts

```diff
@@ -0,0 +1,25 @@
+import { Package } from "frontity/types";
+import Router from "@frontity/router/types";
+import Source from "@frontity/source/types";
+
+/**
+ * Package to do e2e testing of Frontity's rendering.
+ */
+interface Render extends Package {
+  /**
+   * Root components exposed by this package.
+   */
+  roots: {
+    /**
+     * Render namespace.
+     */
+    render: React.ElementType;
+  };
+}
+
+/**
+ * All packages used internally by the Render package.
+ */
+export type Packages = Render & Router & Source;
```

Comment:
We're not using this syntax anymore when we want to define an intersection of packages. Use `MergePackages` instead (from `frontity/types`).

```suggestion
export type Packages = MergePackages<Render, Router, Source>;
```